assets/xml/overlays/ovl_Oceff_Spot.xml

```diff
@@ -0,0 +1,10 @@
+<Root>
+    <File Name="ovl_Oceff_Spot" BaseAddress="0x80972680" RangeStart="0x720" RangeEnd="0xDF8">
+        <Texture Name="sTex" OutName="sun_song_effect" Format="i8" Width="32" Height="32" Offset="0x720"/>
```

Comment:
```suggestion
        <Texture Name="sSunSongEffectTex" OutName="sun_song_effect" Format="i8" Width="32" Height="32" Offset="0x720"/>
```
If you've given it a name, then the texture name should match that

Reply:
Thank you, as a side note, this is mostly a copy paste from OoT, so you might want to fix that there too.

Reply:
Thanks for the heads up!